resources/shuttle-commands.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "Shuttle Commands"
+---
+
+Before using the shuttle services it is important to get to know its different commands.
+To access the different shuttle commands you have to type in the following command in the directory where shuttle was installed:
```

Comment:
This is a good start! I'd move this file to the `introduction` folder in the repo though, and add a link to it by adding it as a page in `mint.json` in the `navigation` section under the `Welcome` group.

I'd also rephrase the second sentence (line 6) to "To get an overview of available commands, run:".